package.json

```diff
@@ -29,6 +31,12 @@
     "rewire-test-helpers": "^0.2.0",
     "rimraf": "^2.4.3",
     "sinon": "^1.17.2",
+    "standard": "^10.0.2",
     "webpack": "^1.12.2"
+  },
+  "standard": {
+    "env": [
```

Comment:
I think it's ok, I doubt that `describe`, `it`, etc. would ever leak to the source code, so it's not a big deal.